Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM centos:8
 
+ARG TARGETARCH
```

Comment:
Ah, that's a cool feature, looks good!

That `TARGETARCH`, can it have a default value, as describe [here](https://docs.docker.com/engine/reference/builder/#arg)?

I'm sure what the value would be, something like `x86_64` or so.

Reply:
Yes, this is an auto-provisioned value by the builder, at least docker will do it. So example values are: `arm64` and `amd64`, for complicated cases, like when the `TARGETPLATFORM` is `linux/arm/v7`, what will be the `TARGETARCH`, I did not test.

Reply:
Since this is an "automatic platform ARG", so its default value will be provided by the docker builder, which defaults to the architecture on which the builder is run.
Please see description on the page you linked: https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope